### PR TITLE
updated .env.sample

### PR DESCRIPTION
**Pull request by dan13ram**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -5,11 +5,25 @@ DATABASE_PASSWORD=metagame_secret
...
+# packages/discord-bot
+GRAPHQL_URL=http://localhost:8080/v1/graphql
+FRONTEND_URL=http://localhost:3000
+DISCORD_BOT_TOKEN=
+DISCORD_BOT_CLIENT_ID=
+DISCORD_BOT_CLIENT_SECRET=
+
+# packages/backend
+IMGIX_TOKEN=
+DISCORD_BOT_TOKEN=  # create one at https://discordapp.com/developers/applications/
```
Comment (alalonde) [inline .env.sample:28]: Is this required for the backend to run? If so, let's fix that. If not, let's just remove this. Contributors shouldn't need to create their own discord app or get the actual token in order to run locally.

File: .env.sample [hunk 2]
```diff
@@ -5,11 +5,25 @@ DATABASE_PASSWORD=metagame_secret
...
 NEXT_BRIGHTID_APP_URL=https://app.brightid.org
 NEXT_BRIGHTID_NODE_URL=http:%2f%2fnode.brightid.org
+NEXT_DISCORD_BOT_CLIENT_ID=808843072980647986
+NEXT_PUBLIC_VERCEL_URL=http://localhost:3000
+
+# packages/discord-bot
+GRAPHQL_URL=http://localhost:8080/v1/graphql
+FRONTEND_URL=http://localhost:3000
+DISCORD_BOT_TOKEN=
+DISCORD_BOT_CLIENT_ID=
```
Comment (alalonde) [inline .env.sample:23]: this is the same as above

**Review by alalonde — CHANGES_REQUESTED** [review]
Remove discord token requirement

**Comment by dan13ram** [comment]
@alalonde I've removed those unneccessary variables as requested :)

File: guides/BACKEND.md [hunk 3]
```diff
@@ -16,6 +16,9 @@ Before you can start the Docker containers, you must run the following.
 # Copy the example environment
 cp .env.sample .env
 
+# Add a github token from https://github.com/settings/tokens to the .env
+echo "GITHUB_API_TOKEN=<github-token>" >> .env
```
Comment (alalonde) [inline guides/BACKEND.md:20]: This code snippet isn't terribly useful as we already have `GITHUB_API_TOKEN=` prefilled

**Review by alalonde — APPROVED** [review]

File: guides/BACKEND.md (hunk 3 above)
Comment (dan13ram) [reply guides/BACKEND.md:20]: Cool. I'm commenting out the line from .env.sample
So that a new builder can learn from either place and add the token.

**Pull request merged**